pwchem/protocols/Sequences/protocol_calculate_conservation.py

```diff
@@ -185,7 +182,7 @@ def calcConsensus(self):
         inFasta = self._getPath('alignedSequences.fasta')
         inSeqs = self.fillInputSequences()
         for seq in inSeqs:
-            seq.appendToFile(inFasta, doClean=False)
+            seq.appendToFile(inFasta) # TODO: add doClean=False in next scipion-em release
```

Comment:
I'd leave this since scipion-em will probably do a release soon, so we do not have to be alert and it would not work on devel this way

Reply:
done